src/App.js

```diff
@@ -1,5 +1,54 @@
+import { Console } from "@woowacourse/mission-utils";
+
 class App {
-  async run() {}
+  async run() {
+    // 문자열 입력
+    Console.print("덧셈할 문자열을 입력해 주세요.");
+    const input = await Console.readLineAsync("");
```

Comment:
Console.readLineAsync에 인자로 들아가는 문자열이 print의 역할을 하고 있어요! 입력 전에 그 입력에 대한 설명을 하도록 하는 것 같아요. `node_modules/@woowacourse/src/console.js` 의 `readLineAsync` 메서드와 `rl.question`의 검색결과를 보면 도움이 될 것 같아요!
```suggestion
    const input = await Console.readLineAsync("덧셈할 문자열을 입력해 주세요.");
```
이런 식으로 바꿀 수 있을 것 같아요!